typing/ctype.mli

```diff
@@ -249,8 +249,14 @@ val expand_head_opt: Env.t -> type_expr -> type_expr
     optimisations. *)
 
 val full_expand: Env.t -> type_expr -> type_expr
+
+type typedecl_extraction_result =
+  | Typedecl of Path.t * Path.t * type_declaration
+  | Has_no_typedecl
+  | May_have_typedecl
+
 val extract_concrete_typedecl:
-        Env.t -> type_expr -> Path.t * Path.t * type_declaration
+        Env.t -> type_expr -> typedecl_extraction_result
         (* Return the original path of the types, and the first concrete
            type declaration found expanding it.
            Raise [Not_found] if none appears or not a type constructor. *)
```

Comment:
Is this comment still accurate?  Looks like it raises less (maybe not at all?)

Reply:
It is not. I'll move it to the type definition and reword it.